nodewatcher/modules/monitor/http/survey/processors.py

```diff
@@ -0,0 +1,160 @@
+import datetime
+import pytz
+
+from django.utils.translation import gettext_noop
+
+from nodewatcher.core.monitor import models as monitor_models, processors as monitor_processors
+from nodewatcher.utils import ipaddr
+from nodewatcher.modules.monitor.sources.http import processors as http_processors
+
+DATASTREAM_SUPPORTED = False
+try:
+    from nodewatcher.modules.monitor.datastream import fields as ds_fields, models as ds_models
+    from nodewatcher.modules.monitor.datastream.pool import pool as ds_pool
+
+    class SurveyInfoStreams(ds_models.RegistryRootStreams):
+        snr = ds_fields.IntegerField(tags={
+            'title': gettext_noop("SNR"),
+            'description': gettext_noop("Node SNR."),
+            'visualization': {
+                'type': 'line',
+                'initial_set': True,
+                'time_downsamplers': ['mean'],
+                'value_downsamplers': ['min', 'mean', 'max'],
+                'minimum': 0.0,
+            },
+        })
+        neighbor_count = ds_fields.IntegerField(tags={
+            'title': gettext_noop("Neighbor count"),
+            'description': gettext_noop("Number of detected access points."),
+            'visualization': {
+                'type': 'line',
+                'initial_set': True,
+                'time_downsamplers': ['mean'],
+                'value_downsamplers': ['min', 'mean', 'max'],
+                'minimum': 0.0,
+            },
+        })
+
+        def get_module_name(self):
+            return 'monitor.http.survey'
+
+    class SurveyInfoStreamsData(object):
+        def __init__(self, node, frequency, channel, snr, neighbors):
+            self.node = node
+            self.frequency = frequency  # either 2 or 5, corresponding to 2.4GHz and 5GHz
+            self.channel = channel
+            self.snr = snr
+            self.neighbors = neighbors
```

Comment:
Doesn't this line store the neighbors JSON object? The neighbor_count variable is only there for plotting the number of neighbors.